app/src/main/java/com/nilhcem/blenamebadge/ui/message/MessagePresenter.kt

```diff
@@ -50,4 +51,8 @@ class MessagePresenter {
             }
         }
     }
+
+    fun showToast(context: Context, message: String) {
```

Comment:
Don't use toast in presenter. I mistook activity for presenter. Revert to previous behaviour

Reply:
I've reverted it to previous code. Now please merge it.